server/notification-providers/signl4.js

```diff
@@ -0,0 +1,85 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+const { Liquid } = require("liquidjs");
+const { UP, DOWN } = require("../../src/util");
+
+class SIGNL4 extends NotificationProvider {
+
+    name = "SIGNL4";
+
+    /**
+     * @inheritdoc
+     */
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        let okMsg = "Sent Successfully.";
+
+        try {
+            let data = {
+                heartbeat: heartbeatJSON,
+                monitor: monitorJSON,
+                msg,
+            };
+            let config = {
+                headers: {
+                    "Content-Type": "application/json"
+                }
+            };
+
+            // Source system
+            data["X-S4-SourceSystem"] = "UptimeKuma";
+
+            // Monitor URL
+            let monitorUrl;
+            if (monitorJSON) {
+                if (monitorJSON.type === "port") {
+                    monitorUrl = monitorJSON.hostname;
+                    if (monitorJSON.port) {
+                        monitorUrl += ":" + monitorJSON.port;
+                    }
+                } else if (monitorJSON.hostname != null) {
+                    monitorUrl = monitorJSON.hostname;
+                } else {
+                    monitorUrl = monitorJSON.url;
+                }
+            }
+
+            if (heartbeatJSON == null) {
```

Comment:
cert-expiry also triggers this route as far as I know.
How does this not show that behaviour in the screenshot? Are the screenshots wrong or am I misremembering this?

Reply:
Hm, something was wrong here. With my latest version. I get certificate alerts like this.
![image](https://github.com/user-attachments/assets/2d54af77-2ab0-4cba-a4d0-5dfa9dc05068)